main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for user, info in data.items():
+        nickname = user
+        email = info["email"] if info["email"] else None
```

Comment:
You can use "info.get(email)" instead and if no email the `None` will be returned by default